reviewing/dependencies-hierarchy/test/getTree.test.ts

```diff
@@ -398,6 +398,60 @@ describe('getTree', () => {
       ])
     })
 
+    test('height === requestedDepth + 1', () => {
```

Comment:
This is similar to the existing `height > requestedDepth` test, but tests the case when `height` is just one more than the requested depth.

This new test fails before the changes in this pull request.